mimesis/providers/personal.py

```diff
@@ -99,49 +96,52 @@ def surname(self, gender: Gender = 0) -> str:
         if self.locale in SURNAMES_SEPARATED_BY_GENDER:
             gender = check_gender(gender)
             return self.random.choice(
-                surnames.get(gender),
+                surnames.get(gender.value),
             )
 
         return self.random.choice(surnames)
 
-    def last_name(self, gender: Gender = 0) -> str:
+    def last_name(self, gender: Gender = Gender.RANDOM) -> str:
+        """An alias of self.surname.
+
+        :param gender: Gender's enum object.
+        :return: Last name.
+        """
         return self.surname(gender)
 
-    def title(self, gender: Gender = 0,
-              title_type: str = 'typical') -> str:
+    def title(self, gender: Gender = Gender.RANDOM,
```

Comment:
Or maybe `Gender.ANY`?

Reply:
`RANDOM` seems the best choice to be. As it clearly states how the gender is selected. How `ANY` gender is selected?